ui.apps/src/main/content/jcr_root/apps/etoolbox-authoring-kit/allowed-children/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't like that the clientlibs folder is missing. It looks like "allowed children" is something truly big, the "size" of "depends-on" or "samples" or "install"
@all, can we just create "clientlibs" and mode DependsOn there, or are there technical considerations?